### PR TITLE
rename actual_nse -> nse_net

### PR DESCRIPTION
**Pull request by zingale**
This is part of the clean up of the 2 different NSE approaches we have.

Depends on #1003

### CODE REVIEW

**Review by zhichen3 — COMMENTED** [review]
still need to fix calling `init_actual_nse()` inside `nse_solver/make_table/main.cpp` . Also correct the error message inside `nse_solver.H` to call `init_nse_net()` when uninitialized, line 60. Perhaps also change the header guard in `nse_solver.H`.

**Comment by zingale** [comment]
I think I fixed them

File: nse_solver/nse_solver.H [hunk 1]
```diff
@@ -17,7 +17,7 @@
```
Comment (zhichen3) [inline nse_solver/nse_solver.H:17]: Do we want to change `_actual_nse_H` to something like `_nse_solver_H` ?

Comment (zingale) [reply nse_solver/nse_solver.H:17]: yeah, I'll change that

Comment (zingale) [reply nse_solver/nse_solver.H:17]: done

**Review by zhichen3 — APPROVED** [review]

**Pull request merged**